extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -84,34 +82,65 @@ export class AmpAnalytics extends AMP.BaseElement {
      * format string used by the tag to send data
      */
     this.requests_ = {};
-    this.element.setAttribute('aria-hidden', 'true');
 
-    if (this.hasOptedOut_()) {
-      // Nothing to do when the user has opted out.
-      log.fine(this.getName_(), 'User has opted out. No hits will be sent.');
-      return Promise.resolve();
-    }
+    /**
+     * @private {JSONObject}
+     */
+    this.remoteConfig = {};
+
+    return this.fetchRemoteConfig_().then(() => {
+      /**
+       * @private {!JSONObject} The analytics config associated with the tag
+       */
+      this.config_ = this.mergeConfigs_();
+
+      if (this.hasOptedOut_()) {
```

Comment:
Just a general comment: Should this be in a more central place, so it isn't as easy to forget?